### PR TITLE
Add tools image for CI runs

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description

Add fleet-manager-tools build to github actions. This image is used in gitlab pipelines to validate gitops YAML files.

## Checklist (Definition of Done)

- [ ] ~~Unit and integration tests added~~
- [ ] ~~Added test description under `Test manual`~~
- [ ] ~~Documentation added if necessary (i.e. changes to dev setup, test execution, ...)~~
- [ ] CI and all relevant tests are passing
- [ ] ~~Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`~~
- [ ] ~~Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.~~
- [ ] ~~Add secret to app-interface Vault or Secrets Manager if necessary~~
- [ ] ~~RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)~~
- [ ] ~~Check AWS limits are reasonable for changes provisioning new resources~~

## Test manual

 - run CI job in gitops gitlab repo

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -549,9 +549,23 @@ image/build/multi-target/probe:
...
+image/push/fleet-manager-tools: image/build/fleet-manager-tools
+	DOCKER_CONFIG=${DOCKER_CONFIG} $(DOCKER) push $(IMAGE_REF)
+	@echo
+	@echo "Image fleet-manager tools was pushed as $(IMAGE_REF)."
+.PHONY: image/push/fleet-manager-tools
+
 # build binary and image and tag image for local deployment
 image/build/local: GOOS=linux
-image/build/local: IMAGE_REF="$(external_image_registry)/$(image_repository):$(image_tag)"
+image/build/local: IMAGE_REF="$(external_image_registry)/rhacs-eng/$(image_repository):$(image_tag)"
```
Comment (ludydoo) [inline Makefile:568]: why this change? (addition of `/rhacs-eng/`)

File: build_push_fleet_manager_tools.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
```
Comment (kurlov) [inline build_push_fleet_manager_tools.sh:1]: Will we built this image in the CI?

Also, we already have almost identical `build_push` scripts for FM/FSS and Probe services. It would be nice to refactor it to use a single file in the future

**Review by kurlov — APPROVED** [review]
🎉

File: build_push_fleet_manager_tools.sh (hunk 2 above)
Comment (SimonBaeumer) [reply build_push_fleet_manager_tools.sh:1]: Add a separate PR: https://github.com/stackrox/acs-fleet-manager/pull/1262

File: Makefile (hunk 1 above)
Comment (SimonBaeumer) [reply Makefile]: Removed, also added PR to remove `image/build/local`: https://github.com/stackrox/acs-fleet-manager/pull/1258
